infrastructure/main.tf

```diff
@@ -133,3 +147,42 @@ resource "azurerm_key_vault_secret" "flyway_password" {
   key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
 }
 # endregion
+
+# region NEW DB secrets
+resource "azurerm_key_vault_secret" "REFORM_POSTGRES_USER" {
+  name         = "${var.component}-REFORM-POSTGRES-USER"
```

Comment:
can we do smth about those names please? and follow some nice guidance:

- lower case
- resource names with underscores
- vault secret names with `-`

i suggest instead `REFORM-POSTGRES` just use `db` - no need to know what db is it. normally we all know what values are needed for db connection: host, port, username, password, db name

Reply:
these are existing names  just reform prefix added.

Reply:
resource names are still capital case

Reply:
existing ones are capital , shall we create another pr to fix them ?

Reply:
existing ones will be removed once migration is complete. they were copied and ignored suggestions